accessibility/README.md

```diff
@@ -1,16 +1,57 @@
-
 # Accessibility Tools
 
 These scripts are designed to be used to complete an isochrone based accessibility analysis for site selection, coverage, catchment area or microplanning analysis.
 
+Read more about why Mapbox built these tools and how to use them in this blog post
+
 ## Installation
 
-With [conda](https://docs.conda.io/en/latest/) installed, set up an environment with the following
+With [conda][1] installed, set up an environment with the following
```

Comment:
why no link to conda @yuletide

Reply:
Link is there -- this is husky enforcing the docs markdown styling rules